README.md

```diff
@@ -1,250 +1,59 @@
-```
-
-```bash
-export SETTINGS__FILE__BASED__SETTINGS__ENV1="machine wins"
-```
-
-```ruby
-puts Settings.file.based.setting.env1
-
-machine wins
-```
-
-Any `Machine based env variables settings` that is not prefixed with `SETTINGS`.\* are not considered for general consumption.
-
-### Documentation
+![Build](https://github.com/DFE-Digital/publish-teacher-training/workflows/Build/badge.svg)
+[![View performance data on Skylight](https://badges.skylight.io/status/NXAwzyZjkp2m.svg?token=JaYZey50Y8gfC00RvzkcrDz5OP-SwiBSTtbhkMw1KIs)](https://www.skylight.io/app/applications/NXAwzyZjkp2m)
 
-Use the following command to generate OpenAPI specification:
+## Table of Contents
 
-```sh
-bundle exec rake rswag:specs:swaggerize
-```
+- [Environments](#environments)
+- [Guides](#guides)
+- [License](#license)
 
-We use [Tech Docs](https://github.com/alphagov/tech-docs-gem) to build documentation. To update documentation, the relevant files can be found in `/docs`. The docker build will then take these files to generate the static site.
+## Environments
 
-To develop and preview the tech docs you can start and run with [Middleman](https://github.com/middleman/middleman)
+### New Find (while the migration is in progress)
 
-```sh
-cd docs && bundle install && bundle exec middleman
-```
+| Name        | URL                                                                    | Description
+| ----------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------------------
+| Production  | [www](https://www2.find-postgraduate-teacher-training.service.gov.uk)   | Public site
+| Staging     | [staging](https://staging2.find-postgraduate-teacher-training.service.gov.uk)| For internal use by DfE to test deploys
+| QA          | [qa](https://qa2.find-postgraduate-teacher-training.service.gov.uk)     | For internal use by DfE for testing. Automatically deployed from main
 
-## CI variables
+### Publish
 
-You'll need to define the `AZURE_CR_PASSWORD` in Travis in order to successfully build and publish. This can be done using this command:
+| Name        | URL                                                                | Description
+| ----------- | ------------------------------------------------------------------ | ---------------------------------------------------------------------
+| Production  | [www](https://www.publish-teacher-training-courses.service.gov.uk) | Public site
+| Staging     | [staging](https://staging.publish-teacher-training-courses.service.gov.uk) | For internal use by DfE to test deploys
+| QA          | [qa](https://qa.publish-teacher-training-courses.service.gov.uk)  | For internal use by DfE for testing. Automatically deployed from main
 
-```bash
-travis encrypt AZURE_CR_PASSWORD="xxx" --add
-```
 
-## Sentry
+## Guides
 
-To track exceptions through Sentry, configure the `SENTRY_DSN` environment variable:
+- [Configuration](/docs/configuration.md)
```

Comment:
What do you think about organising these files into a folder called "guides"?

Reply:
I wish there was a way we could just click on those links 🙃

Reply:
Good idea. You should be able to on Github and locally in your editor? (if it supports it)